### DISCUSSION TITLE
記事の更新中にインジケータ（くるくる）を表示してほしい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#165] [要望・アイデア・不具合]
少し前に記事の投稿や更新時の UI が変わりましたが、更新中にインジケータが表示されず、画面も暗くなりません。
そのため、更新ボタンが押されたのかわかりづらく、画面が操作できるように見えてしまいます。

更新中は画面全体を暗くして操作不能にし、画面中央に更新中を表すインジケータを表示すると親切だと思いました。
もしくは更新を一瞬で終わらせる…笑

**Comment by kskwtnk** [comment]
@uhooi 
ご意見ありがとうございます！

そうですね……私でも時々「あれ、今ってちゃんと押せたかな？」と不安になってしまうタイミングがあります 🙏 
理想としては更新を一瞬で終わらせたいですが、インジケーターなどで現在の状態をフィードバックできるようにします！

**Comment by ohakutsu** [comment]
@uhooi 

おまたせしました！
記事投稿・更新中にアニメーションが表示されるようにしました！

https://qiita.com/release-notes/2023-08-10

**Reply by uhooi** [reply]
おお…ありがとうございます！
